pkg/daemon/ceph/osd/remove.go

```diff
@@ -31,6 +32,13 @@ import (
 	"github.com/rook/rook/pkg/operator/k8sutil"
 )
 
+// CrashFindIdResult is go representation of the Ceph crash ls find command output
+type CrashFindIdResult struct {
```

Comment:
This is not what `cpeh crash ls` gives me, see:

```json
[
    {
        "backtrace": [
            "(()+0x12dd0) [0x7f6028b6cdd0]",
            "(pthread_cond_wait()+0x1fc) [0x7f6028b6848c]",
            "(std::condition_variable::wait(std::unique_lock<std::mutex>&)+0x10) [0x7f60281b68c0]",
            "(AsyncMessenger::wait()+0x1ef) [0x7f602b3276df]",
            "(main()+0x55fb) [0x55a81029f62b]",
            "(__libc_start_main()+0xf3) [0x7f60277c16a3]",
            "(_start()+0x2e) [0x55a8102c8f0e]"
        ],
        "ceph_version": "15.2.7",
        "crash_id": "2021-01-13T15:54:10.692363Z_ad025966-ed16-424e-9236-30350736adab",
        "entity_name": "mon.a",
        "os_id": "centos",
        "os_name": "CentOS Linux",
        "os_version": "8 (Core)",
        "os_version_id": "8",
        "process_name": "ceph-mon",
        "stack_sig": "2a31fa65a650777a15845b7c557d0a172fa949a4ca9cea6da934856404748ad9",
        "timestamp": "2021-01-13T15:54:10.692363Z",
        "utsname_hostname": "rook-ceph-mon-a-6b94b486f8-rlstb",
        "utsname_machine": "x86_64",
        "utsname_release": "4.19.107",
        "utsname_sysname": "Linux",
        "utsname_version": "#1 SMP Thu Mar 26 11:33:10 PDT 2020"
    },
    {
        "backtrace": [
            "(()+0x12dd0) [0x7f98d5ceddd0]",
            "(pthread_cond_wait()+0x1fc) [0x7f98d5ce948c]",
            "(std::condition_variable::wait(std::unique_lock<std::mutex>&)+0x10) [0x7f98d51208c0]",
            "(AsyncMessenger::wait()+0x1ef) [0x7f98d7b0c6df]",
            "(MgrStandby::main(std::vector<char const*, std::allocator<char const*> >)+0x18) [0x55c46ada10c8]",
            "(main()+0x279) [0x55c46ad9ff19]",
            "(__libc_start_main()+0xf3) [0x7f98d472b6a3]",
            "(_start()+0x2e) [0x55c46ada4bae]"
        ],
        "ceph_version": "15.2.7",
        "crash_id": "2021-01-15T13:43:24.004755Z_db6bb3d3-a0cb-4f60-81b2-f7e813de3d66",
        "entity_name": "mgr.a",
        "os_id": "centos",
        "os_name": "CentOS Linux",
        "os_version": "8 (Core)",
        "os_version_id": "8",
        "process_name": "ceph-mgr",
        "stack_sig": "61cd6af5e9d5d3ec279018c55f6769ef12aa74a1ca6f606d63a7be37984aed06",
        "timestamp": "2021-01-15T13:43:24.004755Z",
        "utsname_hostname": "rook-ceph-mgr-a-65d96db875-f5sf4",
        "utsname_machine": "x86_64",
        "utsname_release": "4.19.107",
        "utsname_sysname": "Linux",
        "utsname_version": "#1 SMP Thu Mar 26 11:33:10 PDT 2020"
    }
]
```

Also, you should remove the capital letters.

Reply:
![ceph_health_detail](https://user-images.githubusercontent.com/25681264/105037598-38334880-5a84-11eb-8283-3b4ac1134fe8.png)
https://github.com/rook/rook/issues/5959#issuecomment-668457739
https://ceph.io/community/new-in-nautilus-crash-dump-telemetry/

The output you have shared looks like info of crash, I just want crash ls.

Reply:
Maybe my previous comment got lost but the output is coming from `ceph crash ls`. So the `CrashList` still looks incorrect to me.

Reply:
done.